jenkins-pipelines/manager/older-enterprise-ami-manager-sanity.jenkinsfile

```diff
@@ -7,7 +7,7 @@ managerPipeline(
     backend: 'aws',
     region: '''["us-east-1", "us-west-2"]''',
 
-    scylla_version: '2021.1',
+    scylla_version: '2022.1',
```

Comment:
latest enterprise is `2022.2`

Reply:
Right, but these jobs are ment to run with the second to latest enterprise, hence 2022.1

Reply:
why? IMHO we could run it with the latest released enterprise... only for upgrade purpose i would understand it

Reply:
Maybe just remove those excess jobs ?

Since all now would run with enterprise.

And what about master jobs ? They gonna be triggered with enterprise:next ?

Reply:
> why? IMHO we could run it with the latest released enterprise... only for upgrade purpose i would understand it

That was what @roydahan requested. And I understand, since we are suppose to support both the latest versions

> Maybe just remove those excess jobs ?
> 
> Since all now would run with enterprise.
> 
> And what about master jobs ? They gonna be triggered with enterprise:next ?

Master jobs run with the same version. It's been a long while since we decided to stop running the manager master jobs with scylla master.